screencastr.thor

```diff
@@ -35,6 +37,22 @@ class Screencastr < Thor
   desc "transcode VIDEO_PATH", "Transcode video file to mp4 format"
   def transcode(video_path)
     video = FFMPEG::Movie.new(video_path)
-    video.transcode(FileHelpers.out_path(video_path))
+
+    options = {
+      resolution: "1920x1080", frame_rate: 30
+    }
+
+    video.transcode(FileHelpers.out_path(video_path), options)
+  end
+
+  desc "concat FIRST_VIDEO SECOND_VIDEO", "Concatenate two video files together"
+  def concat(first_video, second_video)
+    out = FileHelpers.concat_out_path(first_video, second_video)
+
+    `ffmpeg -i #{first_video} -i #{second_video} \
+    -filter_complex '[0:v] scale=1920:1080 [vs0]; [1:v] scale=1920:1080 [vs1]; \
```

Comment:
Using `filter-complex` may be overly complicated (I went down this path too). The 90% case is that the videos you'll be joining will have the same encoding, so using `filter_complex` might add slowness to the encoding pipeline for little in the way of benefit.

It's super fast when you do a straight copy. Thoughts?

Reply:
On the other hand, since it's already done and it works, why mess with it, I guess.

Reply:
Ideally we would try simple copy, and then fall back on transcoding if it fails. Like you say though, this is already done, so we should probably add that after the fact.